src/live_restore/live_restore.h

```diff
@@ -12,12 +12,22 @@
 #define WT_LIVE_RESTORE_IN_PROGRESS 0x1
 #define WT_LIVE_RESTORE_COMPLETE 0x2
 
+/*
+ * __wt_live_restore_fh_meta --
+ *     File handle metadata persisted to the WiredTiger metadata file.
+ */
+struct __wt_live_restore_fh_meta {
+    char *bitmap_str;
+    uint64_t bitmap_size;
+    uint32_t allocsize;
+};
+
 /* DO NOT EDIT: automatically built by prototypes.py: BEGIN */
 
-extern int __wt_live_restore_fh_extent_to_metadata(WT_SESSION_IMPL *session, WT_FILE_HANDLE *fh,
-  WT_ITEM *extent_string) WT_GCC_FUNC_DECL_ATTRIBUTE((warn_unused_result));
-extern int __wt_live_restore_fh_import_extents_from_string(WT_SESSION_IMPL *session,
-  WT_FILE_HANDLE *fh, const char *extent_str) WT_GCC_FUNC_DECL_ATTRIBUTE((warn_unused_result));
+extern int __wt_live_restore_fh_import_bitmap(WT_SESSION_IMPL *session, WT_FILE_HANDLE *fh,
```

Comment:
Currently with these function names this call stack reads:
`live_restore_decode_bitmap`
`wt_live_restore_fh_import_bitmap`

and the inverse being
`live_restore_encode_bitmap`
`wt_live_restore_fh_to_metadata`

If I'm reading this correctly, we should align the names `__wt_live_restore_fh_import_bitmap` and `__wt_live_restore_fh_to_metadata`

Reply:
Is the suggestion to change the order of these in this file? I can't do that as that is automatically set by prototypes.py.

Reply:
I'm suggesting to have the names match
`__wt_live_restore_fh_to_metadata` -> `__wt_live_restore_fh_export_bitmap`
or 
`__wt_live_restore_fh_import_bitmap` -> `__wt_live_restore_fh_from_metadata`

Reply:
How does: __wt_live_restore_fh_to_metadata and __wt_live_restore_metadata_to_fh work for you sean?

Reply:
yeah that sounds good!